app/src/page/firstRun/internal/email-verify-page.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Lbry } from 'lbry-redux';
+import {
+  ActivityIndicator,
+  Linking,
+  NativeModules,
+  Platform,
+  Switch,
+  Text,
+  TextInput,
+  View
+} from 'react-native';
+import AsyncStorage from '@react-native-community/async-storage';
+import Button from 'component/button';
+import Colors from 'styles/colors';
+import Constants from 'constants';
+import Icon from 'react-native-vector-icons/FontAwesome5';
+import firstRunStyle from 'styles/firstRun';
+
+class EmailVerifyPage extends React.PureComponent {
+  onResendPressed = () => {
+    const { email, notify, resendVerificationEmail } = this.props;
+    resendVerificationEmail(email);
+    AsyncStorage.setItem(Constants.KEY_EMAIL_VERIFY_PENDING, 'true');
```

Comment:
When/why do we use Async storage vs. Redux?

Reply:
Async storage is used for tracking simple values which don't need to be used more than two or three times when transitioning between screens. Using the redux state would require having to create an action, reducer and selector, which would be unnecessary for the scale of the operation being performed.